people/danny-safdie.md

```diff
@@ -0,0 +1,7 @@
+Danny Safdie
+-------------
+
+![Danny Safdie's photo](photos/danny-safdie.jpg)
```

Comment:
The picture file name doesn't match the actual file name, so the image is broken.